src/pages/EditReportFieldPage.tsx

```diff
@@ -65,11 +65,7 @@ function EditReportFieldPage({route, policy, report}: EditReportFieldPageProps)
                 shouldEnableMaxHeight
                 testID={EditReportFieldPage.displayName}
             >
-                <FullPageNotFoundView
-                    shouldShow
-                    onBackButtonPress={() => {}}
-                    onLinkPress={() => {}}
-                />
+                <FullPageNotFoundView shouldShow />
```

Comment:
Let's use FullPageNotFoundView as a wrapper.

Reply:
I'll update soon

Reply:
@DylanDylann I think our purpose of early returning `FullPageNotFoundView` here is to handle cases where `report`, `reportField`... is undefined, so the `page not found` will be shown immediately. And we did the same in several places within the app like [here](https://github.com/Expensify/App/blob/6a191c52c64b0d6e562c95d6a5241729922df5ca/src/pages/ReimbursementAccount/ReimbursementAccountPage.tsx#L411), [here](https://github.com/Expensify/App/blob/6a191c52c64b0d6e562c95d6a5241729922df5ca/src/pages/settings/Profile/Contacts/ContactMethodDetailsPage.tsx#L163), ...